### PR TITLE
Rename WIDOW to SURVIVING_SPOUSE filing status

### LINKED ISSUE #4094: Rename `WIDOW` to `SURVIVING_SPOUSE` filing status
This is how IRS now describes them

### PR DESCRIPTION
**Pull request by JudyZ13**
Fixes #4094

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: I think this got mixed up with the DC files. Could you take a look and remove all DC related files? You need to check out the PR from your master branch

File: policyengine_us/tests/policy/baseline/gov/states/dc/tax/income/dc_disability_exclusion.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/dc/tax/income/dc_disability_exclusion.yaml:1]: delete file

File: policyengine_us/variables/gov/states/dc/tax/income/subtractions/dc_disability_exclusion.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/dc/tax/income/subtractions/dc_disability_exclusion.py:1]: delete

File: policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/amount.yaml:1]: delete

File: policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/cap.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/cap.yaml:1]: delete

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (PavelMakarchuk) [inline CHANGELOG.md:1]: revert here

File: changelog.yaml [hunk 1]
```diff
@@ -6543,7 +6543,7 @@
 - bump: patch
   changes:
     fixed:
-    - Limit WIDOW filing status to widowed head with child dependents.
+    - Limit suriving spouse filing status to widowed head with child dependents.
```
Comment (PavelMakarchuk) [inline changelog.yaml:6546]: revert here

File: policyengine_us/parameters/gov/contrib/congress/wftca/bonus_guaranteed_deduction/amount.yaml [hunk 2]
```diff
@@ -19,11 +19,11 @@ JOINT:
     2000-01-01: 0
   metadata:
     label: WFTCA joint filer amount
-WIDOW:
+SURVIVING_SPOUSE:
   values:
     2000-01-01: 0
   metadata:
-    label: WFTCA widow filer amount
+    label: WFTCA SURVIVING_SPOUSE filer amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/contrib/congress/wftca/bonus_guaranteed_deduction/amount.yaml:26]: the labels and descriptions can remain unadjusted

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by PavelMakarchuk** [comment]
also need to resolve merge conflicts

File: policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml [hunk 3]
```diff
@@ -2,9 +2,11 @@ description: California provides a renter credit of this amount.
 SINGLE:
   1998-01-01: 60
 SEPARATE:
-  1998-01-01: 60
-WIDOW:
-  1998-01-01: 120
+  2021-01-01: 60
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml:5]:
```suggestion
  1998-01-01: 60
```

File: policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml [hunk 4]
```diff
@@ -2,9 +2,11 @@ description: California provides a renter credit of this amount.
...
+  2022-01-01: 60
+SURVIVING_SPOUSE:
+  2021-01-01: 120
+  2022-01-01: 120
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml:9]:
```suggestion
SURVIVING_SPOUSE:
  1998-01-01: 120
```

File: policyengine_us/parameters/gov/states/ca/tax/income/deductions/itemized/limit/agi_threshold.yaml [hunk 5]
```diff
@@ -24,8 +24,7 @@ JOINT:
 HEAD_OF_HOUSEHOLD:
   2021-01-01: 318_437
   2022-01-01: 344_867
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/deductions/itemized/limit/agi_threshold.yaml:26]: add 2023 value back

File: policyengine_us/parameters/gov/states/ca/tax/income/deductions/standard/amount.yaml [hunk 6]
```diff
@@ -24,8 +24,7 @@ JOINT:
 HEAD_OF_HOUSEHOLD:
   2021-01-01: 9_606
   2022-01-01: 10_404
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/deductions/standard/amount.yaml:26]: add 2023 value back

File: policyengine_us/parameters/gov/states/ca/tax/income/exemptions/phase_out/increment.yaml [hunk 7]
```diff
@@ -2,9 +2,11 @@ description: California reduces its exemption for each of these increments of in
 SINGLE:
   1991-01-01: 2_500
 SEPARATE:
-  1991-01-01: 1_250
-WIDOW:
-  1991-01-01: 2_500
+  2021-01-01: 1_250
+  2022-01-01: 1_250
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/exemptions/phase_out/increment.yaml:6]: revert to old date structure

File: policyengine_us/parameters/gov/states/ca/tax/income/exemptions/phase_out/start.yaml [hunk 8]
```diff
@@ -8,9 +8,7 @@ SEPARATE:
   1991-01-01: 100_000
   2021-01-01: 212_288
   2022-01-01: 229_908
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/tax/income/exemptions/phase_out/start.yaml:10]: add 2023 value

File: policyengine_us/parameters/gov/states/ma/tax/income/credits/senior_circuit_breaker/eligibility/max_income.yaml [hunk 9]
```diff
@@ -14,8 +14,7 @@ HEAD_OF_HOUSEHOLD:
   2001-01-01: 50_000
   2021-01-01: 78_000
   2022-01-01: 80_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ma/tax/income/credits/senior_circuit_breaker/eligibility/max_income.yaml:16]: add 2023 values back

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]
please double check that we didnt delete any lines or changed the dates

**Comment by PavelMakarchuk** [comment]
@MaxGhenis the basic rules PR is done but do we need to change anything in the cps etc.? No tests fail currently

**Review by PavelMakarchuk — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -369,7 +369,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- Limit WIDOW filing status to widowed head with child dependents.
+- Limit widow filing status to widowed head with child dependents.
```
Comment (MaxGhenis) [inline CHANGELOG.md:372]: undo here and in changelog.yaml (we never modify these)

File: policyengine_us/variables/household/demographic/tax_unit/filing_status.py [hunk 11]
```diff
@@ -6,7 +6,7 @@ class FilingStatus(Enum):
     JOINT = "Joint"
     SEPARATE = "Separate"
     HEAD_OF_HOUSEHOLD = "Head of household"
-    WIDOW = "Widow(er)"
+    SURVIVING_SPOUSE = "Widow(er)"
```
Comment (MaxGhenis) [inline policyengine_us/variables/household/demographic/tax_unit/filing_status.py:9]:
```suggestion
    SURVIVING_SPOUSE = "Surviving spouse"
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**